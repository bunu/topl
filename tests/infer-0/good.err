Capturing in javac mode...
Found 2 source files to analyze in /home/rg/research/code/topl/tests/infer-0/infer-out
Starting analysis...

legend:
  "F" analyzing a file
  "." analyzing a procedure

FF.........................................
Found 1 issue

Main.java:18: warning: PRECONDITION_NOT_MET
  in call to `go()` at line 18.
  16.       y.go();
  17.       x.go();
  18. >     x.go();
  19.     }
  20.   }


