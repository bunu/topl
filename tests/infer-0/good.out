Main.java:18: WARNING: PRECONDITION_NOT_MET in call to `go()` at line 18.
